sui-execution/latest/sui-verifier/Cargo.toml

```diff
@@ -15,7 +15,7 @@ move-core-types.workspace = true
 move-vm-config.workspace = true
 move-abstract-stack.workspace = true
 
-move-abstract-interpreter = { path = "../../../external-crates/move/crates/move-abstract-interpreter" }
+move-abstract-interpreter = { workspace = true }
 move-bytecode-verifier = { path = "../../../external-crates/move/crates/move-bytecode-verifier" }
```

Comment:
Is there an issue updating this?

Reply:
Yes there is -- nothing inside `sui-execution` should have its dependencies grouped into workspace dependencies.